src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -49,6 +52,17 @@ public function setDebug(bool $debug)
         $this->debug = $debug;
     }
 
+    public function setTwig(Environment $twig)
+    {
+        $this->twig = $twig;
+        $this->setStrictVariables($twig->isStrictVariables());
+    }
+
+    public function setStrictVariables(bool $strictVariables)
```

Comment:
this should be called in some explicit opt-in way - I suppose via some configuration option